sequencesender/sequencesender.go

```diff
@@ -100,12 +100,68 @@ func (s *SequenceSender) tryToSendSequence(ctx context.Context, ticker *time.Tic
 
 	// Send sequences to L1
 	sequenceCount := len(sequences)
-	log.Infof(
-		"sending sequences to L1. From batch %d to batch %d",
-		lastVirtualBatchNum+1, lastVirtualBatchNum+uint64(sequenceCount),
-	)
+	log.Infof("sending sequences to L1. From batch %d to batch %d", lastVirtualBatchNum+1, lastVirtualBatchNum+uint64(sequenceCount))
```

Comment:
Do we need to add +1 when determining target batch number as well maybe?
```suggestion
	log.Infof("sending sequences to L1. From batch %d to batch %d", lastVirtualBatchNum+1, lastVirtualBatchNum+1+uint64(sequenceCount))
```